<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardSenderForm</class>
 <widget class="QWidget" name="CardSenderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="sendtextEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetPort">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>GetPort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CloseDevice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>InitDevice</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxComm">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SendCard</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="receivetextEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>发送数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>接收数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetStatus">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>GetStatus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
